### ISSUE TITLE
Standalone vctrs?

### ISSUE THREAD

**Issue by JosiahParry** [R-ArcGIS/arcgisutils#33]
Follow up with: https://github.com/R-ArcGIS/arcgislayers/issues/58
Possible solution from gabor is to use vctrs standalone from r-lang 
https://fosstodon.org/@gaborcsardi/111097312300288758

would like to have tibble-esque printing with data.frame class. The tables we get are fairly nasty.

**Comment by elipousson** [comment]
Would love to see improved printing for data frames from arcgislayers! One note though: vctrs is already a dependency via dbplyr and httr2. Why not just add it to your imports?

**Comment by JosiahParry** [comment]
Good point! Standalone is nice because then upstream changes won't cause errors here. It's like vendoring dependencies which has its benefits! 

The standalone version adds `tbl` class but `vctrs::data_frame()` does not and the `tbl` class is the pretty pillar part :)
